assembly/src/parsers/serde/deserialization.rs

```diff
@@ -129,18 +129,41 @@ impl<'a> ByteReader<'a> {
 /// Returns `self` from its byte representation stored in provided `ByteReader` struct.
 pub trait Deserializable: Sized {
     fn read_from(bytes: &mut ByteReader) -> Result<Self, SerializationError>;
+
+    fn try_from_bytes(bytes: &[u8]) -> Result<Self, SerializationError> {
```

Comment:
nit: I would probably rename this to `read_from_bytes()` as the only thing different between this one and `read_from()` is this one takes byte slice as a parameter.

Reply:
Fixed!